### DISCUSSION TITLE
Directus docker image is outdated / how to update

### DISCUSSION THREAD

**Discussion by MariuszKogut** [directus/directus#17533] [Archive]
Hey guys,

I'm wondering why the official docker image (https://hub.docker.com/r/directus/directus) is out of date. 

How can I update the docker image inside the docker container?
And can you please establish a process to make sure that every release is also applied to the docker image?

Thank you very much,
Mariusz

**Comment by br41nslug** [comment]
The docker image is not out of date so no action required :)

**Reply by MariuszKogut** [reply]
Well. Getting this warning:

``
2023-02-15 14:16:46 [13:16:45.909] WARN: Update available: 9.22.1 -> 9.22.4
``

**Reply by br41nslug** [reply]
Interesting then you're not running the latest tag it seems, are you using `image: directus/directus:9.22.4` as per the latest tag? https://hub.docker.com/r/directus/directus/tags

If you're running `directus/directus:latest` you may want to pull in the latest image using `docker pull`

**Reply by MariuszKogut** [reply]
Yeah - you are awesome! 🥳

Using the tag did the trick. 

``
image: directus/directus:9.22.4
``

I am sorry for my inability. 🙈

**Reply by kevincam3** [reply]
When we update by changing the directus version this way, do we need to run any script to migrate the database or is that check already handled when changing to a newer directus docker image?

**Reply by paescuj** [reply]
This is automatically handled on startup of the container 👍
